chrome-extension/utils/plugins/make-manifest-plugin.ts

```diff
@@ -1,49 +1,57 @@
-import fs from 'node:fs';
+import { copyFileSync, existsSync, mkdirSync, writeFileSync } from 'node:fs';
 import { resolve } from 'node:path';
 import { pathToFileURL } from 'node:url';
-import process from 'node:process';
+import { env, platform } from 'node:process';
+import type { Manifest } from '@extension/dev-utils';
 import { colorLog, ManifestParser } from '@extension/dev-utils';
 import type { PluginOption } from 'vite';
-import type { Manifest } from '@extension/dev-utils/dist/lib/manifest-parser/type';
 
-const rootDir = resolve(__dirname, '..', '..');
-const refreshFile = resolve(__dirname, '..', 'refresh.js');
-const manifestFile = resolve(rootDir, 'manifest.js');
+const manifestFile = resolve(import.meta.dirname, '..', '..', 'manifest.js');
+const refreshFile = resolve(
+  import.meta.dirname,
+  '..',
+  '..',
+  '..',
+  'packages',
+  'hmr',
+  'dist',
+  'lib',
+  'injections',
+  'refresh.js',
+);
```

Comment:
Different between `chrome-extension/utils/refresh.js`

Reply:
It doesn't exist

Reply:
I've move it like it's before to `packages/hmr/lib/injections/refresh.ts`